src/components/SettingsComponent.vue

```diff
@@ -65,7 +65,7 @@
       }
     },
     methods: {
-      submit(e) {
+      submit() {
         console.log(this.newPlayer)
```

Comment:
No I just noticed the submit function was taking in a statement but doing nothing with is so I removed it. I will make a pull request shortly for a cleaned up master without all the console.logs.